<?xml version="1.0"?>
<launch>
  
# Things to Know!
# input_velocity_topic_name will write to           -- geometry_msgs::Twist
# output_velocity_topic_name will write             -- geometry_msgs::Twist
# linear_velocity_limit will write                  -- double (max velocity in m/s)
# angular_velocity_limit will write                 -- double (max velocity in rad/s)


	<!-- <arg name="input_velocity_topic_name"         default="/arm/joint_group_velocity_controller/command"/>
	<arg name="output_velocity_topic_name"        default="/UR10arm/desired_joint_velocity"/> -->
	<arg name="input_velocity_topic_name"         default="/UR10arm/desired_joint_velocity"/>
	<arg name="output_velocity_topic_name"        default="/arm/joint_group_velocity_controller/command"/>
	<arg name="joint_velocity_limit"              default="0.85"/>

	<group ns="jointds_filter">
		<param name="input_velocity_topic_name"   value="$(arg input_velocity_topic_name)"/>
		<param name="output_velocity_topic_name"  value="$(arg output_velocity_topic_name)"/>
		<param name="joint_velocity_limit"        value="$(arg joint_velocity_limit)"/>
		<node name="cddynamics_joint_filter_node" pkg="online_motion_generator" type="cddynamics_joint_filter_node" output="screen" />
	</group>

</launch>